packages/deploy/deploy/500_marketplace/504_deploy_exchange.ts

```diff
@@ -43,7 +43,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 export default func;
 func.tags = ['Exchange', 'Exchange_deploy'];
 func.dependencies = [
-  'RoyaltiesRegistry_deploy',
-  'OrderValidator_deploy',
+  'RoyaltiesRegistry',
+  'OrderValidator',
   'TRUSTED_FORWARDER_V2',
```

Comment:
Actually these looked correct to me before, not sure why you changed them?
  'RoyaltiesRegistry_deploy',
  'OrderValidator_deploy',

Reply:
the deploy tags weren't calling `OrderValidator_setup` and `RoyaltiesRegistry_setup`